### PR TITLE
vlang: Initial implementation

### PR DESCRIPTION
**Pull request by Kreyren**
Fixes: https://github.com/gitpod-io/gitpod/issues/1143

Credit: @uniminin for help with the import

Signed-off-by: Jacob Hrbek <kreyren@rixotstudio.cz>

### CODE REVIEW

**Comment by Kreyren** [comment]
Disclaimer: as referenced on https://github.com/gitpod-io/gitpod/issues/1143 this is apparently expected to run on root.

**Comment by Kreyren** [comment]
Ready for review

**Comment by Kreyren** [comment]
The symlink to `v` executable fails with this implementation, fixing..

File: full/Dockerfile [hunk 1]
```diff
@@ -253,6 +253,17 @@ RUN cp /home/gitpod/.profile /home/gitpod/.profile_orig && \
 
 RUN bash -lc "cargo install cargo-watch"
 
+### VLANG ###
+USER root
+ENV VLANG_SOURCE="/opt/vlang"
+RUN git clone https://github.com/vlang/v "$VLANG_SOURCE" || exit 1
```
Comment (RDIL) [inline full/Dockerfile]: Maybe you should use a stable release instead of nightly?

Comment (Kreyren) [reply full/Dockerfile]: rafactored, better version selection adapted

**Comment by JesterOrNot** [comment]
I feel that Vlang is far too unstable to add to the default workspace image I wouldn't feel comfortable maybe in a separate workspace image, although it might be nice for the Vlang language guide 🤔

**Pull request closed without merging**